github/api.go

```diff
@@ -83,12 +96,25 @@ func (a *Client) LatestRelease(repo string) (*Release, error) {
 // Releases for a particular repo.
 func (a *Client) Releases(repo string) (releases []Release, err error) {
 	url := fmt.Sprintf("https://api.github.com/repos/%s/releases", repo)
-	return releases, a.decode(url, &releases)
+	// Paginate.
+	for n := 1; n < 100; n++ {
+		var page []Release
+		url = fmt.Sprintf("%s?per_page=100&page=%d", url, n)
+		err = a.decode(url, &page)
+		if err != nil {
+			return nil, err
+		}
+		releases = append(releases, page...)
+		if len(page) < 100 {
+			break
```

Comment:
Maybe we should log a warning if we never break here?

Reply:
I made it an error.